package.json

```diff
@@ -16,7 +16,8 @@
     "scripts": {
         "preinstall": "npx only-allow pnpm",
         "lint": "eslint .",
-        "test": "pnpm jest --passWithNoTests"
+        "test": "pnpm jest --passWithNoTests",
+        "format": "prettier --write \"packages/**/*.ts\" \"apps/**/*.ts\""
```

Comment:
format command has been added

Reply:
I have added prettier formatter in eslint configuration.
So, I think we can use ``eslint --fix .`` but I need to check if it works as expected.